modules/installation-create-ingress-dns-records.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-aws.adoc
+
+[id="installation-create-ingress-dns-records_{context}"]
+= Creating the Ingress DNS Records
+
+If you removed the DNS Zone configuration, manually create DNS records that point to the Ingress load balancer.
+You can create either a wildcard record or specific records. While the following procedure uses A records, you can use other record types that you require, such as CNAME or alias.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Install the `jq` package.
+
+.Procedure
+
+. Determine the routes to create.
+** To create a wildcard record, use `*.apps.<cluster_name>.<domain_name>`, where `<cluster_name>` is your cluster name, and `<domain_name>` is the Route53 base domain for your {product-title} cluster.
+** To create specific records, you must create a record for each route that your cluster uses, as shown in the output of the following command:
++
+----
+$ oc get --all-namespaces -o jsonpath='{range .items[*]}{range .status.ingress[*]}{.host}{"\n"}{end}{end}' routes
+
+oauth-openshift.apps.<cluster_name>.<domain_name>
+console-openshift-console.apps.<cluster_name>.<domain_name>
+downloads-openshift-console.apps.<cluster_name>.<domain_name>
+alertmanager-main-openshift-monitoring.apps.<cluster_name>.<domain_name>
+grafana-openshift-monitoring.apps.<cluster_name>.<domain_name>
+prometheus-k8s-openshift-monitoring.apps.<cluster_name>.<domain_name>
+----
+
+. Retrieve the Ingress load balancer status and note the value of the `EXTERNAL-IP` address that it uses:
++
+----
+$ oc -n openshift-ingress get service router-default
+NAME             TYPE           CLUSTER-IP      EXTERNAL-IP                            PORT(S)                      AGE
+router-default   LoadBalancer   172.30.62.215   ab3...28.us-east-2.elb.amazonaws.com   80:31499/TCP,443:30693/TCP   5m
+----
+
+. Locate the hosted zone ID for the load balancer:
++
+----
+$ aws elb describe-load-balancers | jq -r '.LoadBalancerDescriptions[] | select(.DNSName == "<external_ip>").CanonicalHostedZoneNameID' <1>
+
+Z3AADJGX6KTTL2
+----
+<1> For `<external_ip>`, specify the value of the external IP address of the Ingress Operator load balancer that you obtained.
++
+The output of this command is the load balancer hosted zone ID.
+
+. Obtain the public hosted zone ID for your cluster's domain:
++
+----
+$ aws route53 list-hosted-zones-by-name \
+            --dns-name "<domain_name>" \ <1>
+            --query "HostedZones[? Config.PrivateZone != \`true\` && Name == \`<domain_name>.\`].Id" \ <1>
```

Comment:
if you aren't using variable substitution for the domain name, like:

    Name == \`${BASE_DOMAIN}.\`

you can use single quotes which lets you avoid escaping the backticks:

    --query 'HostedZones[? Config.PrivateZone != `true` && Name == `<domain_name>.`].Id'